src/Entity/MenuItem.php

```diff
@@ -17,7 +17,8 @@
  *      indexes={
  *          @ORM\Index(columns={"name", "path"}),
  *          @ORM\Index(columns={"lft", "rgt", "root"}),
- *          @ORM\Index(columns={"root", "lft"})
+ *          @ORM\Index(columns={"root", "lft"}),
+ *          @ORM\Index(columns={"lvl", "language"})
```

Comment:
Is dit niet iets dat ook nog in eerdere versies gexit kan worden en dan forward mergen? Dan profiteren Sf4 sites er ook nog van bij de volgende composer update...

Reply:
Ja was wel aardig geweest, maar de winst is maar zo minimaal. Ik heb het niet gedaan omdat de sites langzaam zijn maar om het ene openstaande issue dat er nog is te kunnen wegwerken :grin: